docs/UserGuide.md

```diff
@@ -1,172 +1,191 @@
+        - 5.2.3 [Delete a partner course: `partnercourse delete`](#523-delete-a-partner-course-partnercourse-delete)
+        - 5.2.4 [Update a partner course: `partnercourse update`](#524-update-a-partner-course-partnercourse-update)
+        - 5.2.5 [Search for a partner course by an attribute: `partnercourse search`](#525-search-for-a-partner-course-by-an-attribute-partnercourse-search)
+      - 5.2.6 [Sort all partner courses by an attribute: `partnercourse sort`](#526-sort-all-partner-courses-by-an-attribute-partnercourse-sort)
+    - 5.3 [Commands for universities](#53-commands-for-universities)
+        - 5.3.1 [List all universities: `university list`](#531-list-all-universities-university-list)
+        - 5.3.2 [Search for a university by an attribute: `university search`](#532-search-for-a-university-by-an-attribute-university-search)
+      - 5.3.3 [Sort all universities by an attribute: `university sort`](#533-sort-all-universities-by-an-attribute-university-sort)
+    - 5.4 [Commands for mappings](#54-commands-for-mappings)
+        - 5.4.1 [List all mappings: `mapping list`](#541-list-all-mappings-mapping-list)
+        - 5.4.2 [Add a mapping: `mapping add`](#542-add-a-mapping-mapping-add)
+        - 5.4.3 [Delete a mapping: `mapping delete`](#543-delete-a-mapping-mapping-delete)
+        - 5.4.4 [Search for a mapping by an attribute: `mapping search`](#544-search-a-mapping-by-attributes-mapping-search)
+        - 5.4.5 [Sort all mappings by an attribute: `mapping sort`](#545-sort-a-mapping-by-attributes-mapping-sort)
+    - 5.5 [Commands for notes](#55-commands-for-notes)
+        - 5.5.1 [List all notes: `note list`](#551-list-all-notes-note-list)
+        - 5.5.2 [Add a note: `note add`](#552-add-a-note-note-add)
+        - 5.5.3 [Delete a note: `note delete`](#553-delete-a-note-note-delete)
+        - 5.5.4 [Update a note: `note update`](#554-update-a-note-note-update)
+        - 5.5.5 [Tag a note: `note tag`](#555-tag-a-note-note-tag)
+        - 5.5.6 [Clear all tags from a note: `note cleartag`](#556-clear-all-tags-in-a-note-note-cleartag)
+    - 5.6 [View help : `help`](#56-view-help--help)
+    - 5.7 [Exit SEPlendid: `exit`](#57-exit-seplendid--exit)
+    - 5.8 [Save the data](#58-save-the-data)
 6. [FAQ](#6-faq)
 7. [Command summary](#7-command-summary)
 
 </div>
-
+<br>
+<br>
 --------------------------------------------------------------------------------------------------------------------
 
 ## 1. How to navigate this User Guide?
 
-As a new user in SEPlendid, this user guide serves as an easy-to-follow guide in aiding you in executing your first 
-commands in SEPlendid, utilising SEPlendid to it's fullest potential!
+As a new user in SEPlendid, this user guide serves as an easy-to-follow guide, guiding you through the execution of your
+first commands in SEPlendid. You can unlock SEPlendid full potential in this guide!
+
+Here is a step-by-step instruction for you to navigate the user guide:
```

Comment:
Are these changes really necessary? Please credit the initial writer, you only changed a few words.

Reply:
I was trying to improve the "you" language as feedbacked by dr wee during our cs2101 tutorial. If you deem it uncessary, I shall revert

Reply:
I agree, and also had followed Dr. Wee's kind recommendations. No need to revert. I meant above as a suggestion that you should credit the original writer if you are using a large chunk of their work. This follows after I seen that files such as `MappingAttributeEnum` (and the local/partner course equivalents) were replaced, without proper credit. Just a small note.